src/renderer/components/AgentExchangeList.vue

```diff
@@ -0,0 +1,92 @@
+<template >
```

Comment:
This just looks like the connection list copied and renamed?

Reply:
Probably shouldn't include this until you've made changes that actually need to be committed.